osbenchmark/telemetry.py

```diff
@@ -409,48 +415,49 @@ def record(self):
         # pylint: disable=import-outside-toplevel
         import elasticsearch
 
-        try:
-            ccr_stats_api_endpoint = "/_ccr/stats"
-            filter_path = "follow_stats"
-            stats = self.client.transport.perform_request("GET", ccr_stats_api_endpoint, params={"human": "false",
-                                                                                                 "filter_path": filter_path})
-        except elasticsearch.TransportError:
-            msg = "A transport error occurred while collecting CCR stats from the endpoint [{}?filter_path={}] on " \
-                  "cluster [{}]".format(ccr_stats_api_endpoint, filter_path, self.cluster_name)
-            self.logger.exception(msg)
-            raise exceptions.BenchmarkError(msg)
+        for index in self.indices:
+            try:
+                stats = self.client.transport.perform_request("GET", "/_plugins/_replication/" + index + "/_status")
```

Comment:
Should we consider integrating `leader_stats` and `follower_stats` as well?

Reply:
Added.